RELEASE.md

```diff
@@ -0,0 +1,61 @@
+# How to make a release
+
+`jupyterhub-tmpauthenticator` is a package available on [PyPI].
+
+These are the instructions on how to make a release.
+
+## Pre-requisites
+
+- Push rights to this GitHub repository
+
+## Steps to make a release
+
+1. Create a PR updating `CHANGELOG.md` with [github-activity] and continue when
+   its merged.
+
+   Advice on this procedure can be found in [this team compass
+   issue](https://github.com/jupyterhub/team-compass/issues/563).
+
+2. Checkout main and make sure it is up to date.
+
+   ```shell
+   git checkout main
+   git fetch origin main
+   git reset --hard origin/main
+   ```
+
+3. Update the version, make commits, and push a git tag with `tbump`.
+
+   ```shell
+   pip install tbump
+   ```
+
+   ```shell
+   # Example versions to set: 1.0.0, 1.0.0b1
+   VERSION=
+
+   tbump --dry-run ${VERSION}
```

Comment:
Since tbump has a confirmation step before proceeding, I'm not sure `--dry-run` is a very helpful step. `tbump ${VERSION}` does the same thing as dry-run, but waiting for your input before actually doing anything.